src/hooks/useContributions.ts

```diff
@@ -33,7 +33,7 @@ export function useContributionUsers(): ContributionUser[] {
             queryKey: `circle-integration-contributions-${i.id}-${epoch.id}`,
             queryFn: () =>
               fetch(
-                `https://api.dework.xyz/integrations/coordinape/${i.data.organizationId}?epoch_start=${epoch.start_date}&epoch_end=${epoch.end_date}`
+                `https://api.deworkxyz.com/integrations/coordinape/${i.data.organizationId}?epoch_start=${epoch.start_date}&epoch_end=${epoch.end_date}`
```

Comment:
One user had issues with his Adblocker blocking dework.xyz, but letting deworkxyz.com (our other domain) through. We've had similar issues with people on our website, so we set up a .com that doesn't get blocked